packages/web/src/app/doc-pages/brand/illustrations-doc/illustrations-exhibit/illustrations-exhibit-details/illustrations-exhibit-details.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, inject } from '@angular/core';
+import { takeUntilDestroyed, toSignal } from '@angular/core/rxjs-interop';
+import { filter } from 'rxjs';
+
+import { IllustrationsExhibitService } from '../illustrations-exhibit.service';
+import { IllustrationsGeneratorComponent } from '../illustrations-generator/illustrations-generator.component';
+import { CopyComponent } from 'src/app/shared/copy/copy.component';
+
+@Component({
+  selector: 'app-illustrations-exhibit-details',
+  standalone: true,
+  imports: [CopyComponent, IllustrationsGeneratorComponent],
+  templateUrl: './illustrations-exhibit-details.component.html',
+  styleUrl: './illustrations-exhibit-details.component.scss',
+})
+export class IllustrationsExhibitDetailsComponent {
+  selectedIllustration = toSignal(
+    inject(IllustrationsExhibitService).selectedIllustration.pipe(takeUntilDestroyed(), filter(Boolean)),
+  );
```

Comment:
Do we need the `takeUntilDestroyed()` here? 🧐
Reading [toSignal](https://angular.io/api/core/rxjs-interop/toSignal) it states:

> By default, the subscription will be automatically cleaned up when the current [injection context](https://angular.io/guide/dependency-injection-context) is destroyed. For example, when [toObservable](https://angular.io/api/core/rxjs-interop/toObservable) is called during the construction of a component, the subscription will be cleaned up when the component is destroyed.

Reply:
Probably not no! I think it just came from copilots suggestion, but it can be removed 🧹